Gruntfile.js

```diff
@@ -1,22 +1,35 @@
-/*jshint node:true */
+/* jshint node:true */
```

Comment:
This line can be removed, right? Or updated to not mention jshint.

Reply:
Yes, removed